src/option.README.md

```diff
@@ -0,0 +1 @@
+# Option
```

Comment:
@JamieB-gu I might need your help writing a readme for these modules... do you have time some time? 🙏

Reply:
Sure, happy to! How would you like to do this? I can push to this PR, I can create a PR into this PR, or we can pair?